src/main/java/core/basesyntax/service/impl/CsvReader.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class CsvReader implements ReaderService {
+    private static final int CSV_FORMAT_INDEX_OF_TITLE = 0;

```

Comment:
this class should be reusable, so let's decide in constructor if we have an title and should we remove it or not